tests/test_ArrayDimChecker.cpp

```diff
@@ -247,21 +246,36 @@ BOOST_AUTO_TEST_SUITE(WellDimensions)
 
 BOOST_AUTO_TEST_CASE(MaxGroupSize)
 {
-    auto cse = CaseObjects{ simCaseWellDims() };
+    Opm::ParseContext parseContext;
+    auto cse = CaseObjects{ simCaseWellDims(), parseContext };
 
     // Verify at most ten wells in a single group.
     BOOST_CHECK_EQUAL(Opm::maxGroupSize(cse.sched, 1), 10);
 }
 
 BOOST_AUTO_TEST_CASE(WellDims)
 {
-    auto cse = CaseObjects{ simCaseWellDims() };
+    Opm::ParseContext parseContext;
+    parseContext.update(Opm::ParseContext::RUNSPEC_NUMWELLS_TOO_LARGE, Opm::InputError::THROW_EXCEPTION);
```

Comment:
This sequence of updates overrides the defaults (without changing them), so while the test is fine, the default set is not actually tested?

Reply:
That is correct. Personally I think it is best to split the default the default settings - i.e. *policy* from the testing of the functionality - this is also in line with the other tests of `ParseContext` error handling: https://github.com/OPM/opm-common/blob/master/tests/parser/ParseContextTests.cpp